src/app/modules/webhook/webhook.service.ts

```diff
@@ -221,6 +222,13 @@ export const sendWebhook = (
   })
 }
 
+export const getWebhookType = (webhookUrl: string) => {
+  const isZapier = webhookUrl.match(/^https:\/\/hooks\.zapier\.com\//)
+  const isPlumber = webhookUrl.match(/^https:\/\/plumber\.gov\.sg\/webhooks\//)
```

Comment:
Apologies I am late to make the comment, but better late than never 😅 

* Use `string.match(rexep)` when you are trying to extract part of an input with a regexp
* When all that's needed is boolean check, use `regexp.test(string)` instead.

It's not a big concern, but using match always create a data structure of the match (or `null` if no match), so it's more costly and consumes more memory. `regexp.test()` on the other hand just returns a boolean, and so is as cheap as the check can be.

BTW i you need to extract multiple entries in a regexp, use `regexp.exec(string)`

Reply:
Thanks Tim, changed to `regexp.test(string)` in #6475